dist/unix/systemd/qbittorrent-nox@.service.in

```diff
@@ -0,0 +1,14 @@
+[Unit]
+Description=qBittorrenti-nox service for user %I
+
+Documentation=man:qbittorrent-nox(1)
+After=network.target
+
+[Service]
+User=%i
+ExecStart=@EXPAND_BINDIR@/qbittorrent-nox
+Type=simple
+PrivateTmp=yes
```

Comment:
There is no typo, it is intuitive vocabulary I use in commits names, in Git or on wikis.

Like:
w - for 'write'
rw - for 'rewrite'
del - for 'deletions'
add
mv - for 'move'
sec - for 'section'
subsec - for 'subsection'
and so on.

So:
```
add subsec "anton" in sec "Draft"
rw subsec "anton"
mv subsec "anton" to sec "Name"
w to subsec "anton" useful information about qBittorrent.
```

I don't know why something like this is not widely adopted, so I promote it by example.